Library/Homebrew/test/utils/shell_spec.rb

```diff
@@ -0,0 +1,68 @@
+require "utils/shell"
+
+describe Utils::Shell do
+  describe "::path_to_shell" do
```

Comment:
I've never seen `describe "::method"` used for class methods before — it's usually `describe ".method" in my experience. It looks a bit strange to me since methods aren't typically referred to with `::`.

Thoughts?